sgx/pelz_enclave.edl

```diff
@@ -44,8 +45,44 @@ enclave {
  *
  * @return 0 on success, 1 on error
  */
-	public int key_table_delete(charbuf key_id);
+public int key_table_delete(charbuf key_id);
 
+/**
+ * <pre>
+ * This function initializes a server table for lookup.
+ * <pre>
+ *
+ * @param[in] server_table The pointer for server table to be initialized
```

Comment:
```server_table_init``` doesn't take these parameters.

Reply:
```server_table``` is an argument for the function and is used in the function. The function intakes ```server_table``` as an extern variable so you don't have to call the variable as an input argument which is the same way we do it for the key table.